src/writer/blocks/writeraskkg.py

```
@@ -115,41 +115,42 @@ def run(self):
+                delta_answer = chunk.model_extra.get("answer", "")
+                answer += delta_answer
+
+                if graph_citations:
+                    delta_sources = chunk.model_extra.get("sources", "")
+                    citations.extend(delta_sources)
+                    self._set_state(state_element, {"answer": answer, "citations": citations})
+                else:
+                    self._set_state(state_element, answer)
+
+            except json.JSONDecodeError:
+                logging.error("Could not parse stream chunk from graph.question")
```

Ruff (0.14.14)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)